File: polyglot/seis_native/fsharp/SeisNativeKernel.fs
namespace Seis.NativeKernel

type NativeRoadmapItem =
    { Lane: string
      Score: int
      Language: string }

module SeisNativeKernel =
    let roadmap =
        [ { Lane = "Apple First"; Score = 100; Language = "Swift" }
          { Lane = "Data AI"; Score = 88; Language = "Python" }
          { Lane = "Systems"; Score = 84; Language = "Rust" }
          { Lane = "Android"; Score = 76; Language = "Kotlin" }
          { Lane = "Windows"; Score = 72; Language = "CSharp" }
          { Lane = "Infrastructure"; Score = 70; Language = "Go" } ]

    let topLane = List.head roadmap

File: polyglot/seis_native/erlang/seis_native_kernel.erl
-module(seis_native_kernel).
-export([roadmap/0, top_lane/0]).

roadmap() ->
    [
        #{lane => "Apple First", score => 100, language => "Swift"},
        #{lane => "Data AI", score => 88, language => "Python"},
        #{lane => "Systems", score => 84, language => "Rust"},
        #{lane => "Android", score => 76, language => "Kotlin"},
        #{lane => "Windows", score => 72, language => "CSharp"},
        #{lane => "Infrastructure", score => 70, language => "Go"}
    ].

top_lane() ->
    hd(roadmap()).
